dist/gcc-compatible/Hacl_IntTypes_Intrinsics_128.c

```diff
@@ -0,0 +1,55 @@
+/* MIT License
```

Comment:
this file seems superfluous... was it leftover in your dist directory?

Reply:
I don't think it's superfluous, I split `Hacl.IntTypes.Intrinsics` into 2 files, the uint128-compatible part and the current fallback implementations.

Reply:
but the definitions are also in the .h file right below https://github.com/project-everest/hacl-star/pull/503/files/93bffad7ac99d060e71a00d99995dcf1bf508a6b#diff-a75ea96722ac2fa74fa24a86706cc737abef098e98a605580837a5b138a209c9 so they appear once as `static inline` in the `.h` and once more in the `.c`? if so, that would be a kremlin bug

Reply:
Ah yes you're right and indeed it was a leftover file